From stable-bounces@linux.kernel.org  Tue Aug 30 12:47:40 2005
Date: Tue, 30 Aug 2005 15:47:14 -0400
From: Michael Krufky <mkrufky@m1k.net>
To: stable@kernel.org
Cc: Andrew Morton <akpm@osdl.org>,
        Mauro Carvalho Chehab <mchehab@brturbo.com.br>, torvalds@osdl.org,
        linux-dvb-maintainer@linuxtv.org,
        linux-kernel <linux-kernel@vger.kernel.org>
Subject: [PATCH] Kconfig: saa7134-dvb must select tda1004x

I wish I had seen this before 2.6.13 was released... I guess this only 
goes to show that there haven't been any testers using saa7134-hybrid 
dvb/v4l boards that depend on the tda1004x module, during the 2.6.13-rc 
series :-(

Please apply this to 2.6.14, and also to 2.6.13.1 -stable.  Without this 
patch, users will have to EXPLICITLY select tda1004x in Kconfig.  This 
SHOULD be done automatically when saa7134-dvb is selected.  This patch 
corrects this problem.

saa7134-dvb must select tda1004x

Signed-off-by: Michael Krufky <mkrufky@m1k.net>
Signed-off-by: Chris Wright <chrisw@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/media/video/Kconfig |    1 +
 1 files changed, 1 insertion(+)

Index: linux-2.6.13.y/drivers/media/video/Kconfig
===================================================================
--- linux-2.6.13.y.orig/drivers/media/video/Kconfig
+++ linux-2.6.13.y/drivers/media/video/Kconfig
@@ -254,6 +254,7 @@ config VIDEO_SAA7134_DVB
 	select VIDEO_BUF_DVB
 	select DVB_MT352
 	select DVB_CX22702
+	select DVB_TDA1004X
 	---help---
 	  This adds support for DVB cards based on the
 	  Philips saa7134 chip.
